src/public/index.html

```diff
@@ -9,37 +9,54 @@
   </head>
 
 
-    <body>
-      <header>
-        <h1>Something Awesome!</h1>
-      </header>
-      <nav>
-        <ul>
-          <li><a href="/about/">About</a></li>
-          <li> <a href="/contact/">Contact</a></li>
-        </ul>
-      </nav>
+  <body>
+    <header>
+      <div class="title">
+        <h1 class="plaintext-color">
+            Something Awesome! <br/>
+        </h1>
+      </div>
+    </header>
+      <div class="content-container page">
+        <div class="nav-width">
+          <nav class = 'red'>
+            <ul>
+              <li>
+                <a href="/about/" class='hoverlink-color'>About</a>
+              </li>
+              <li>
+                <a href="/contact/" class='hoverlink-color'>Contact</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
 
-      <main>
-        <h2> Latest Posts: </h2>
+      <div class="main-width">
+        <main class = 'yellow'>
```

Comment:
It's fine to use background colors as a visual guide when developing the layout, but don't commit them.